tests/baselines/reference/escapedReservedCompilerNamedIdentifier.js

```diff
@@ -38,7 +38,7 @@ var o = {
 };
 var b = o["__proto__"];
 var o1 = {
-    __proto__: 0
+    __proto__: {}
 };
 var b1 = o1["__proto__"];

```

Comment:
Accessing the `__proto__` property like this should be an error, unless `__proto__` is defined on the global `Object` interface or the `__proto__` property is declared using `{ ["__proto__"]: ... }`.

Reply:
Yes, as you can see some of my changes reflected that:

```
tests/cases/conformance/es6/modules/m2.ts(2,21): error TS2339: Property '__proto__' does not exist on type '{ __esmodule: boolean; }'.
```